script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -61,17 +61,17 @@ Mutated monsters like you, for example."
 "Wha... what are you talking about? A vampire like you has no right to call me a monster!"
 "A monster is a monster. Even among vampires, there are none that possess the Mystic Eyes that can see the death of things."
 <Page19>
-"...? The death, of things...?"
+"The death... of things...?"
 "What else could it be?
 Shiki. Your eyes must have a connection to a place that should not be connected to any living thing.... Are those eyes something you were born with?"
 "...No, they've been like this for a long time, but they're not innate."
 <Page20>
 "In that case, you must have experienced death at least once before, right?"
 "I'm not like you, so it's not like I can just simply die.
-...But yes, I have. I almost died in an accident once. It was only after that that I began to see those lines."
+...But yes, I have. I almost died in an accident once. It was only after that when I began to see those lines."//to be rechecked
```

Comment:
After that was when I began to see those lines.

Reply:
It was only after that incident that I began to see those lines.

IIRC original rephrase intent was to avoid `that that`